### PR TITLE
Change couchbase DB to storage DB

### PR DESCRIPTION
**Pull request by mdonaka**
# Summary
Update to use storage DB

# Purpose
Save memory and handle large data.

# Contents
- Update CC to use ComputationToDb
- Update MC to use ManageToDb
- Fix piece_id on MC to use 0-index
- Fix sendModelParam on MC to create completed file

# Testing Methods Performed
- make test t=LibClient p=medium
- Run hjoin request with large data
- CI

### CODE REVIEW

File: src/ComputationContainer/Job/JobStatus.hpp [hunk 1]
```diff
@@ -3,8 +3,8 @@
 #include <boost/format.hpp>
 #include <optional>
 #include <stdexcept>
-
-#include "Client/ComputationToDbGate/Client.hpp"
+;
```
Comment (arukuka) [inline src/ComputationContainer/Job/JobStatus.hpp:6]: Cloud you remove semi-colon?

File: src/ManageContainer/Client/ManageToDb/Client_test.go [hunk 2]
```diff
@@ -176,7 +176,7 @@ func TestGetComputationResultSuccess(t *testing.T) {
 	initialize()
 
 	os.Mkdir(fmt.Sprintf("/Db/result/%s", defaultJobUUID), 0777)
-	data := "{\"id\":\"\",\"job_uuid\":\"m2db_test_jobuuid\",\"result\":\"[\\\"1\\\",\\\"2\\\",\\\"3\\\"]\",\"meta\":{\"piece_id\":1}}"
+	data := "{\"id\":\"\",\"job_uuid\":\"m2db_test_jobuuid\",\"result\":\"[\\\"1\\\",\\\"2\\\",\\\"3\\\"]\",\"meta\":{\"piece_id\":0}}"
```
Comment (arukuka) [inline src/ManageContainer/Client/ManageToDb/Client_test.go:179]: Could you take this opportunity to use raw string literal: `` ` ``?

https://go.dev/ref/spec#String_literals

It is okay to add another refactoring task, and do it at the time.

**Review by arukuka — CHANGES_REQUESTED** [review]
Thank you for your contribution!
I commented on the part that I was curious about, so please check it.

File: src/ComputationContainer/Job/JobStatus.hpp (hunk 1 above)
Comment (mdonaka) [reply src/ComputationContainer/Job/JobStatus.hpp]: I remove it in ac007d5bf85cb16e4942e0cda18aaa3aa71af9b8!

File: src/ManageContainer/Client/ManageToDb/Client_test.go (hunk 2 above)
Comment (mdonaka) [reply src/ManageContainer/Client/ManageToDb/Client_test.go]: Thank you! You've been very helpful!
I have updated with this PR in 088a14d401a844d9eda7662b259a6528148cd19d as there are few changes.

File: src/ComputationContainer/Job/JobStatus.hpp (hunk 1 above)
Comment (arukuka) [reply src/ComputationContainer/Job/JobStatus.hpp]: Thank you!

File: src/ManageContainer/Client/ManageToDb/Client_test.go (hunk 2 above)
Comment (arukuka) [reply src/ManageContainer/Client/ManageToDb/Client_test.go]: Thank you!  I checked it.

**Review by arukuka — APPROVED** [review]

**Pull request merged**